### PR TITLE
initial translation commit

### PR DESCRIPTION
**Pull request by gluneau**
This is my initial effort to make this project multilingual.  I will either update this pull request with more commits or other pull request if you merge it first.

Great references: 
- https://pythonhosted.org/Flask-Babel/
- https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-xiv-i18n-and-l10n

@robrigo please contact me, I have some questions about AccountExistsException

### CODE REVIEW

File: agoricco/__init__.py [hunk 1]
```diff
@@ -5,16 +5,20 @@
...
 app = Flask(__name__)
+babel = Babel(app)
 
 
 if 'FLASK_APP_SETTINGS' in os.environ:
     app.config.from_envvar('FLASK_APP_SETTINGS')
 else:
     app.config.from_object('agoricco.default_settings')
 
+app.config['BABEL_DEFAULT_LOCALE'] = 'fr'
```
Comment (robrigo) [inline agoricco/__init__.py]: Does this need to be set? Or does `BABEL_DEFAULT_LOCALE` get a default from babel? If it must be set, I would ask that it goes in the `default_settings.py` file.

**Comment by robrigo** [comment]
Nice work! I'll need to pull this locally and play around before I can accept the PR. Feel free to keep updating this branch with your work.

**Comment by gluneau** [comment]
That's it for now @robrigo, please merge and I will submit my contribution to Utopian.io

Thank you.

**Comment by robrigo** [comment]
Nice work, thank you @gluneau !

**Pull request merged**